libs/core/dtos/animeItem.dto.ts

```diff
@@ -0,0 +1,36 @@
+/** Anime list DTO. */
+export interface AnimeItemDto {
+
+	/** ID. */
+	readonly id: number;
+
+	/** Creation date. */
+	readonly created: string;
+
+	/** Modification date. */
+	readonly modified: string;
+
+	/** Title. */
+	readonly title_eng: string;
+
+	/** Image title. */
+	readonly title_jpn: string;
+
+	/** Image URL. */
+	readonly image: string;
+
+	/** Airing tart and end dates. */
```

Comment:
By moving it to a separate model we can shorten description here to
```suggestion
	/** Airing period. */
```